### DISCUSSION TITLE
how to change movement control keys

### DISCUSSION THREAD

**Discussion by Fish154** [mozilla/hubs#3598] [General Questions & Discussion] [answered]
I do not use a qwerty keyboard. How to change the movement control keys from the default WASD ? It's impossible in the user preferences.

**Comment by vincentfretin** [comment, marked as answer]
You can't. It's a known issue, see https://github.com/mozilla/hubs/issues/2397

**Comment by matthewbcool** [comment]
The arrow keys can also be used for movement if that helps.
